se/executables_create_draft.py

```diff
@@ -322,7 +323,7 @@ def _get_wikipedia_url(string: str, get_nacoaf_url: bool) -> (str, str):
 
 	return None, None
 
-def create_draft(args: list):
+def create_draft(args: Namespace):
```

Comment:
If we're changing the type that gets passed here, do we need to change where it's called as well?

Reply:
The callers already pass in argparse.Namespace objects, that's why I changed the annotation.